pkg/plugins/backendplugin/grpcplugin/client_v2.go

```diff
@@ -151,15 +151,15 @@ func (c *ClientV2) CheckHealth(ctx context.Context, req *backend.CheckHealthRequ
 
 func (c *ClientV2) QueryData(ctx context.Context, req *backend.QueryDataRequest) (*backend.QueryDataResponse, error) {
 	if c.DataClient == nil {
-		return nil, backendplugin.ErrMethodNotImplemented
+		return nil, plugins.ErrMethodNotImplemented
 	}
 
 	protoReq := backend.ToProto().QueryDataRequest(req)
 	protoResp, err := c.DataClient.QueryData(ctx, protoReq)
 
 	if err != nil {
 		if status.Code(err) == codes.Unimplemented {
-			return nil, backendplugin.ErrMethodNotImplemented
+			return nil, plugins.ErrMethodNotImplemented
 		}
 
 		return nil, fmt.Errorf("%v: %w", "Failed to query data", err)
```

Comment:
Should this and other methods in here use `plugins.ErrPluginDownstreamError` for consistency now?

Reply:
same, can we change this freely?

Reply:
Yeah that's true